costSplit/index.html

```diff
@@ -1,36 +1,121 @@
+
+        <div class='byPerson'>
+          <label for=people>Number of people<input type='number' id="splitCost" /></label>
+          <button id="btn">Split!!</button>
+          <p id="textField"></p>
+        </div>
+      </div>
+    </section>
+
+    <section>
+      <label for="splitCost1">Split cost by percent</label>
+      <input type="radio" id="splitCost1" name="split-cost" value="byPercent">
+
+      <div class='byPercent'>
+        <div>
+          <input type='text' id="name" placeholder="Name" />
+          <input type='number' id="person1" placeholder="%" />
+        </div>
+        <div>
+          <input type='text' id="name" placeholder="Name" />
+          <input type='number' id="person2" placeholder="%" />
+        </div>
+        <button id='percentBtn'>Split</button>
+        <p class='error'></p>
+        <p id='amount1'></p>
+        <p id='amount2'></p>
+      </div>
+    </section>
+
+  </main>
+  <script>
+    const amountField = document.getElementById("amt");
+    const splitCostField = document.getElementById("splitCost");
+    const button = document.querySelector('button');
+    const textField = document.querySelector("#textField");
+
+    button.addEventListener('click', () => {
+      let amount = Number(amountField.value);
+      let splitCost = Number(splitCostField.value);
+
+      const indvAmt = calculateAmount(amount, splitCost);
+
+      const text = `Every person needs to pay ${indvAmt}`
+      textField.innerHTML = text;
+    });
+
+    const amount1 = document.querySelector('#amount1');
+    const amount2 = document.querySelector('#amount2');
+    const person1 = document.querySelector('#person1');
+    const person2 = document.querySelector('#person2');
+    const radioBtns = document.querySelectorAll('input[type=radio]');
+    const percentBtn = document.querySelector('#percentBtn');
+    const names = document.querySelectorAll('input[id=name]');
+
+
+    const splitPercentage = () => {
+      let amount = +amountField.value;
+      let first = +person1.value / 100;
+      let second = +person2.value / 100;
+      let error = document.querySelector('.error');
+
+      if (first > 1 || second > 1) {
+        return error.innerHTML = 'The percentage entered must be less than 100%. Please try again.';
+      }
+      if (first) {
+        person2.value = 100 - person1.value;
+        second = +person2.value / 100;
+      } else {
+        person1.value = 100 - person2.value;
+        first = +person1.value / 100;
+      }
+      amount1.innerHTML = `${names[0].value || 'Person1'} will pay ${Number(first * amount).toFixed(2)}`;
+      amount2.innerHTML = `${names[1].value || 'Person2'} will pay ${Number(second * amount).toFixed(2)}`;
+    }
+    //onclick will calculate the amount and correct the percentages
+    percentBtn.addEventListener('click', () => {
+      let amount = Number(amountField.value);
+      splitPercentage();
+    })
+
+    const calculateAmount = (amount, splitCost) => {
```

Comment:
We should have a pattern for readability, either move the addEventListener to one place followed by the eventHandler methods or every eventListener followed by its event handler method.